bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BTable.java

```diff
@@ -49,14 +78,27 @@ public Object value() {
 
     @Override
     public String stringValue() {
-        return "";
+        if (tableProvider == null) {
```

Comment:
better if we can use a boolean flag for this condition rather than depending on tableProvider being null